README.md

```diff
@@ -11,37 +11,77 @@ intuitive, and predictable.
 
 ```console
 admin              Administrative commands
-alloc              allocate a new instance for interactive use
+alloc              Interactively enqueue a job within a sub allocation
 attach             Interactively attach to job
+batch              Non-interactively enqueue a job within a sub allocation
```

Comment:
This also isn't quite right - I would say "submit a batch job" and "submit a job" and don't get caught up on the interactive thing.

Reply:
Again, politely I disagree. I think we need to think take special care to define which commands **are** and **aren't** interactive since this'll help users understand the differences between `batch`, `alloc`, `run`, `submit`. Each function does something similar but has different interactive/non-interactive characteristics.

Reply:
Same as above - I'm fine to compromise leaving the focus on interactive / not interactive but "enqueue a job within a sub allocation" would be akin to saying "move one leg in front of the other in a swift motion to propel the body in movement" vs. "take a run." The descriptions need to be more human friendly and readable by even a layperson.